playback/src/player.rs

```diff
@@ -214,21 +214,21 @@ pub fn coefficient_to_duration(coefficient: f64) -> Duration {
 
 #[derive(Clone, Copy, Debug)]
 pub struct NormalisationData {
-    track_gain_db: f32,
-    track_peak: f32,
-    album_gain_db: f32,
-    album_peak: f32,
+    track_gain_db: f64,
+    track_peak: f64,
+    album_gain_db: f64,
+    album_peak: f64,
 }
 
 impl NormalisationData {
     fn parse_from_file<T: Read + Seek>(mut file: T) -> io::Result<NormalisationData> {
         const SPOTIFY_NORMALIZATION_HEADER_START_OFFSET: u64 = 144;
         file.seek(SeekFrom::Start(SPOTIFY_NORMALIZATION_HEADER_START_OFFSET))?;
 
-        let track_gain_db = file.read_f32::<LittleEndian>()?;
-        let track_peak = file.read_f32::<LittleEndian>()?;
-        let album_gain_db = file.read_f32::<LittleEndian>()?;
-        let album_peak = file.read_f32::<LittleEndian>()?;
+        let track_gain_db = file.read_f64::<LittleEndian>()?;
+        let track_peak = file.read_f64::<LittleEndian>()?;
+        let album_gain_db = file.read_f64::<LittleEndian>()?;
+        let album_peak = file.read_f64::<LittleEndian>()?;
```

Comment:
This is the problem, these fields are stored as `f32`. In `new-api` I already converted everything to `f64` because the Symphonia metadata reader can possibly return such metadata for other formats, so perhaps I can spare you the time 😉

Reply:
Yep, I noticed that. Wow that was distorted,lol!!!

Reply:
MY main goal was to make everything `f64` so we don't have to convert stuff all the time. I doubt it cost any CPU but it makes for simpler code.